src/main/events/library/open-game.ts

```diff
@@ -1,16 +1,42 @@
+import { exec as execCallback } from 'child_process';
 import { gameRepository } from "@main/repository";
-
 import { registerEvent } from "../register-event";
 import { shell } from "electron";
+import { promisify } from 'util';
+
+const exec = promisify(execCallback);
+
+const escapeShellPath = (path: string) => {
+  // Escapes spaces, parentheses for commands
+  return path.replace(/ /g, '\\ ').replace(/\(/g, '\\(').replace(/\)/g, '\\)');
+};
 
 const openGame = async (
   _event: Electron.IpcMainInvokeEvent,
   gameId: number,
   executablePath: string
 ) => {
-  await gameRepository.update({ id: gameId }, { executablePath });
+  try {
+    await gameRepository.update({ id: gameId }, { executablePath });
+    await exec(`chmod +x "${executablePath}"`);
```

Comment:
Not an expert, but Isn't this a breaking change in windows? Is this code path executed only on linux?